L1Trigger/TrackFindingTracklet/test/L1TrackNtupleMaker_cfg.py

```diff
@@ -40,27 +41,29 @@
 process.MessageLogger.Tracklet = dict(limit = -1)
 process.MessageLogger.TrackTriggerHPH = dict(limit = -1)
 
+
+print("using geometry " + GEOMETRY + " (tilted)")
+process.load('Configuration.Geometry.GeometryExtendedRun4' + GEOMETRY + 'Reco_cff')
+process.load('Configuration.Geometry.GeometryExtendedRun4' + GEOMETRY +'_cff')
+
+from Configuration.AlCa.GlobalTag import GlobalTag
+# Change needed to run with D98 geometry in recent CMSSW versions.
 if GEOMETRY == "D88" or GEOMETRY == 'D98':
-    print("using geometry " + GEOMETRY + " (tilted)")
-    process.load('Configuration.Geometry.GeometryExtended2026' + GEOMETRY + 'Reco_cff')
-    process.load('Configuration.Geometry.GeometryExtended2026' + GEOMETRY +'_cff')
+    process.GlobalTag = GlobalTag(process.GlobalTag, '133X_mcRun4_realistic_v1', '')
+elif GEOMETRY == 'D110':
+    process.GlobalTag = GlobalTag(process.GlobalTag, 'phase2_realistic', '')
```

Comment:
Need the `auto` part in front
```suggestion
    process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase2_realistic', '')
```